bookwyrm/templates/lists/list.html

```diff
@@ -40,6 +81,16 @@
                             <input type="hidden" name="item" value="{{ item.id }}">
                             <button type="submit" class="button is-small is-danger">{% trans "Remove" %}</button>
                         </form>
+                    </div>
+                    <div class="card-footer has-background-white-bis">
+                        <div>
+                            <form name="set-position" method="post" action="{% url 'list-set-book-position' item.id %}" class="card-footer-item">
+                                {% csrf_token %}
+                                <label for="input-list-position" class="is-sr-only">{% trans "Set list position" %}</label>
```

Comment:
To be more consistent with how form elements are labelled elsewhere, I think "List position" would be more descriptive than "Set list position", since that's the wording on the button.

Reply:
This language should be updated now.